packages/adapters-library/src/scripts/generateAdapter.test.ts

```diff
@@ -92,7 +92,7 @@ describe('generateAdapter function snapshots', () => {
   generateAllOutcomes().forEach((outcomes, index) => {
     it(`should match snapshot for outcomes ${index}`, async () => {
       const blankTemplate = await readBlankTemplate(
-        'packages/adapters-library/src/adapters/blank/blankTemplate/blankAdapterForCli/blankAdapter.ts',
+        'src/adapters/blank/blankTemplate/blankAdapterForCli/blankAdapter.ts',
```

Comment:
did you mean to change that?

Reply:
Does it work?

Reply:
Yes, otherwise the test fails. Jest paths seem to work different.

Reply:
ok saw slack